src/comment.cpp

```diff
@@ -304,7 +304,7 @@ void CommentParser::collectParameters(const char *fulltext, FileModule *root_mod
 
 		// Look for the group to which the given assignment belong
 		int i=0;
-		for (;i<groupList.size() && groupList[i].lineNo<firstLine;i++);
+		for (; (unsigned)i < groupList.size() && groupList[i].lineNo<firstLine;i++);
```

Comment:
I assume this is part of the customiser, which I have never used, so I don't really understand what this does. Your code looks more correct but I can't convince myself it does the same as the original because of the --i. So I think perhaps it either fixes a bug or creates a new one.